packages/core/src/impl/browser/FetchTransport.ts

```diff
@@ -153,20 +157,25 @@ export default class FetchTransport implements Transport {
         }
       } catch (e) {
         Log.warn('Unable to receive error body', e)
+
         throw new HttpError(
           response.status,
           response.statusText,
           undefined,
           response.headers.get('retry-after'),
-          response.headers.get('content-type')
+          response.headers.get('content-type'),
+          undefined,
+          headersToBeReturned
         )
       }
       throw new HttpError(
         response.status,
         response.statusText,
         text,
         response.headers.get('retry-after'),
-        response.headers.get('content-type')
+        response.headers.get('content-type'),
+        undefined,
+        headersToBeReturned
```

Comment:
Please use our build-in function `getResponseHeaders(response)`:

```suggestion
          getResponseHeaders(response)
```